### ISSUE TITLE
Don't hardcode slash commands

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#186487]
Chat accessibility help hardcodes certain slash commands. This is wrong because they are contributed by extensions and can be dynamic. Instead the [Session#session](https://github.com/microsoft/vscode/blob/fb2354b794ff063cb17c21dc6dfdd32d9ce8d544/src/vs/workbench/contrib/inlineChat/browser/inlineChatSession.ts#L111)-object should be used which knows about all slash commands
